src/test/java/seedu/address/testutil/EditPatientDescriptorBuilder.java

```diff
@@ -53,10 +45,9 @@ public EditPatientDescriptorBuilder(Patient patient) {
         descriptor.setBloodType(patient.getBloodType());
         descriptor.setNokName(patient.getNokName());
         descriptor.setNokPhone(patient.getNokPhone());
-        descriptor.setAllergy(patient.getAllergies());
+        //descriptor.setAllergy(patient.getAllergies());
```

Comment:
Might want to elaborate in comments that this is awaiting the switch to using List for the Allergy class